File: SchoolWork/app/src/main/java/com/example/app01/MainActivity.java
package com.example.app01;

import androidx.appcompat.app.AppCompatActivity;

import android.graphics.Color;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView tv3 = findViewById(R.id.tv3);
        tv3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                tv3.setText(tv3.getText() + " 문자추가");
                tv3.setTextColor(Color.RED);
                tv3.setTextSize(30);
            }
        });
    }
}

File: GSMach/app/src/main/java/com/example/gsmach/MainActivity.kt
package com.example.gsmach

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.gsmach.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    val binding by lazy { ActivityMainBinding.inflate(layoutInflater) }
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(binding.root)


    }
}